ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2088,8 +2085,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = org.reactjs.native.example.BetterSocial;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match AdHoc org.reactjs.native.example.BetterSocial";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `PROVISIONING_PROFILE_SPECIFIER` has been removed. This could lead to issues with code signing and app distribution. Ensure that these changes are intentional and correctly managed in your CI/CD pipeline or manually before distribution.
```diff
- PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
+ PROVISIONING_PROFILE_SPECIFIER = "";
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->